graphql_schema.go

```diff
@@ -47,7 +47,7 @@ schema {
 	}
 
 	type PlayerSession {
-		playerId: ID!
```

Comment:
I don't think you should only allow getting a playerid via the player linked entity